cloudinit/sources/DataSourceOracle.py

```diff
@@ -135,19 +143,29 @@ def _get_data(self):
 
         # network may be configured if iscsi root.  If that is the case
         # then read_initramfs_config will return non-None.
-        fetch_vnics_data = self.ds_cfg.get(
-            "configure_secondary_nics",
-            BUILTIN_DS_CONFIG["configure_secondary_nics"],
-        )
         network_context = noop()
-        if not _is_iscsi_root():
+        if _has_run_net_files():
```

Comment:
I don't think that the old `if not _is_iscsi_root()` check was necessary. It can be removed.

Reply:
Removed. Could you please clarify this a bit? I am not sure why this was introduced in the first place, and now it's not required. Thanks.